examples/semi_supervised/plot_label_propagation_digits.py

```diff
@@ -69,15 +73,15 @@ class will be very good.
 print("Confusion matrix")
 print(cm)
```

Comment:
No need to print `cm` or compute the confusion matrix. Instead, we will use a display to show graphically the confusion matrix:

```python
from sklearn.metrics import ConfusionMatrixDisplay

ConfusionMatrixDisplay.from_predictions(
    true_labels, predicted_labels, labels=lp_model.classes_
)
```